crates/red_knot_python_semantic/src/db.rs

```diff
@@ -136,7 +136,7 @@ pub(crate) mod tests {
         /// Target Python platform
         python_platform: PythonPlatform,
         /// Path to a custom typeshed directory
-        custom_typeshed: Option<SystemPathBuf>,
+        typeshed: Option<SystemPathBuf>,
```

Comment:
hmm, it is still true that we will still _have_ a typeshed directory to use even if this field is `None` though, right? (We'll just use the vendored typeshed directory.) In that sense, even if we're renaming the CLI option to `--typeshed` rather than `--custom-typeshed`, I still think `custom_typeshed` might be a better variable name for us to use internally for fields like this.

Reply:
The field is called `typeshed` (with the same type) in `program::SearchPathSettings`, which is used in lots of places. I chose a consistent naming here. I renamed both fields now.

Reply:
It looks like many of these fields have now been renamed from `typeshed` to `custom_typeshed`, but still not this specific one 😆

Reply:
Oops, I'll take a look. Thanks.